src/plot_recipes.jl

```diff
@@ -5,7 +5,7 @@
 @userplot VoxelReusePlot
 
 @recipe function f(vr::VoxelReusePlot; tmin=nothing, tmax=nothing,
-                   overlap=(1/6,1/6,1/6), nreal=10, gpu=false,
```

Comment:
Yes. You're right. I added a resource parameter to the f function and passed it to voxelreuse. But I'm not sure about the safety of this change because I don't know how it affects the callers of the function. I would need to study about @recipe macros to understand it, but I await your feedback on this.